saml_idp.gemspec

```diff
@@ -41,6 +41,7 @@ defaults in a Production environment. Post any issues you to github.
   s.add_dependency('builder')
   s.add_dependency('httparty')
   s.add_dependency('nokogiri')
+  s.add_dependency('xmlenc', '>= 0.6.4')
```

Comment:
We can make it an optional dependency, esp if we split the assertion building into an encryption-specific class, and then load `xmlenc` explicitly in the encrypted assertion class. I'm find with that. Mostly a "install it if you intend to use it" philosophy, with some helpful error messaging.